Cargo.lock

```diff
@@ -7824,7 +7825,7 @@ version = "0.1.9"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "cf221c93e13a30d793f7645a0e7762c55d169dbb0a49671918a2319d289b10bb"
 dependencies = [
- "windows-sys 0.59.0",
+ "windows-sys 0.48.0",
```

Comment:
I can't explain why cargo decided to go to the bottom of the accepted range here instead of the top